packages/webpack/test/e2e-dynamic-import.spec.js

```diff
@@ -0,0 +1,316 @@
+const test = require('ava')
+const { scaffold, runScriptWithSES } = require('./scaffold.js')
+const { makeConfig } = require('./fixtures/main/webpack.config.js')
+
+test.before(async (t) => {
```

Comment:
(suggestion) define the context type ([example](https://github.com/LavaMoat/LavaMoat/blob/68fa29918d76deec44c955c9ac1f085c6a3a906e/packages/node/test/e2e/cli-generate.spec.js#L18-L25))

if you choose _not_ to do this, it's helpful to get actual types for `test`:

```js
/**
 * @import {TestFn} from 'ava'
 */
const test = /** @type {TestFn} */(require('ava')
```

...since the version of `ava` we're using has broken types for CJS; the type assertion is not necessary in ESM.  🙄